src/types.ts

```diff
@@ -45,6 +45,7 @@ export interface NodeFileTraceOptions {
   readFile?: (path: string) => Buffer | string | null;
   stat?: (path: string) => Stats | null;
   readlink?: (path: string) => string | null;
+  resolve?: (id: string, parent: string) => Promise<string | string[]>;
```

Comment:
Does this need to be async? The other options are synchronous and I thought your previous PR was synchronous too?

Reply:
I was confused by the documentation that has `readFile` and `readLink` as promises 🤦